calculator.c

```diff
@@ -984,6 +1018,14 @@ int main(int argc, char *argv[])
 			free(tokens);
 			tokens = NULL;
 			numTokens = 0;
+			for (i = 0; i < nrCalcTokens; i++)
```

Comment:
I tried it but it didn't work out. Moreover, I tried to do as less changes as possible to your code because the version mismatch in LIKWID (BSD vs. GPLv3) requires that I mark all code fragments I included in your code.
But it should be possible to free all tokens that are evaluated in `evalStackPush` and push only the result in a new allocated space on the stack. If I have time, I try that.